website/static/css/style.css

```diff
@@ -337,6 +337,12 @@ a {
     margin-top: 4.8px;
 }
 
+/* Disable pointer events in page changing */
+.disabled {
+    cursor: default !important;
+    pointer-events: none;
```

Comment:
You didn't add this rule, but will it be a problem if it doesn't work? `pointer-events` is supported only in IE 11+.

ref: https://css-tricks.com/almanac/properties/p/pointer-events/